internal/services/user.go

```diff
@@ -108,6 +108,11 @@ func (s *tokenManager) GenerateTokens(ctx context.Context, login, password strin
 	if err != nil {
 		return "", "", true, err
 	}
+
+	if user.IsDeleted {
+		return "", "", false, errors.New("account has been deleted")
```

Comment:
Please consider renaming the error message 'account has been deleted' to 'User deleted' for consistency with the terminology used throughout the system.